pandas/tests/extension/test_arrow.py

```diff
@@ -1676,6 +1677,15 @@ def test_to_numpy_int_with_na():
     tm.assert_numpy_array_equal(result, expected)
 
 
+@pytest.mark.parametrize("na_val, exp", [(lib.no_default, np.nan), (1, 1)])
+def test_to_numpy_null_array(na_val, exp):
```

Comment:
Could you also test the `dtype=None, na_value=lib.no_default` case?

Reply:
done